samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,351 @@
+                innerRadius: '95%'
+            }
+        }, {
+            size: '100%',
+            innerSize: '95%',
+            ...paneOpeningAngles,
+            background: {
+                borderWidth: 16,
+                borderColor: '#F3DEF7',
+                backgroundColor: {
+                    radialGradient: { cx: 0.5, cy: 0.5, r: 1.8 },
+                    stops: [
+                        [0, 'rgba(66, 72, 179,1)'],
+                        [0.2, 'rgba(232, 201, 245, 1)'],
+                        [0.25, 'rgba(243, 255, 240, 1)'],
+                        [0.3, 'rgba(10, 255, 226, 1)']
+                    ]
+                },
+                innerRadius: '55%',
+                outerRadius: '100%'
+            }
+        }],
+
+        xAxis: [{
+            pane: 0,
+            lineWidth: 8,
+            lineColor: '#8575BF',
+            tickInterval: 1,
+            gridLineWidth: 0,
+            min: 1,
+            max: 26,
+            labels: { enabled: false }
+        }, {
+            pane: 1,
+            linkedTo: 0,
+            gridLineWidth: 0,
+            lineWidth: 4,
+            lineColor: '#708FFF',
+            plotBands: Array(3).fill(7).map(
+                (weekendOffset, week) => {
+                    const
+                        from = weekendOffset * (week + 1),
+                        to = from - 1;
+                    return {
+                        from,
+                        to,
+                        color: '#FF7A70'
+                    };
+                }
+            ),
+            ...monthExtremes,
+            labels: { enabled: false }
+        }, {
+            pane: 2,
+            tickAmount: 5,
+            tickInterval: 1,
+            gridLineWidth: 0,
+            lineColor: '#A69BE8',
+            lineWidth: 16,
+            ...weekExtremes,
+            labels: { enabled: false }
+        }],
+
+        yAxis: [{
+            pane: 0,
+            tickInterval: 8,
+            gridLineWidth: 1,
+            max: 1800,
+            min: -8,
+            labels: { enabled: false }
+        }, {
+            pane: 1,
+            reversed: true,
+            gridLineWidth: 0,
+            tickInterval: 100,
+            min: 0,
+            max: 400,
+            labels: { enabled: false }
+        }, {
+            pane: 2,
+            tickInterval: 0.25,
+            gridLineWidth: 4,
+            gridLineColor: '#D8FDF4',
+            min: -3,
+            max: 1,
+            labels: { enabled: false }
+        }],
+
+        legend: {
+            enabled: true,
+            backgroundColor: '#E2F4F8',
+            borderColor: '#D2B8FA',
+            borderRadius: 32,
+            floating: true,
+            layout: 'vertical',
+            verticalAlign: 'top',
+            squareSymbol: true,
+            borderWidth: 4,
+
+            itemStyle: { color: '#333333', fontSize: '0.86rem', letterSpacing: '0.032rem' },
```

Comment:
Empty line and a too long line (shouldn't linter pick this up?).

Reply:
Linter is not counting strings and probably that's why there wasn't any error here.

Reply:
> Linter is not counting strings and probably that's why there wasn't any error here.

Ah I see! I had no idea!